docs/UserGuide.md

```diff
@@ -514,25 +597,26 @@ and with phone number either `12345678` or `87654321`.
   email either `john@example.com` or `alice@example.com`
   * `find ph d/IT r/SWE Manager` returns potential hires with department `IT`, and role either `SWE` or `Manager`
 
-Example: "find ph n/John".
-![result for 'find ph n/John'](images/findjohnResult.png)
+  Example: "find ph n/John".
+  ![result for 'find ph n/John'](images/findjohnResult.png)
```

Comment:
i think the john here needs to be capital J like John for the images/findjohnResult

Reply:
Good find! However, that issue will be resolved in #203 which has yet to be merged